jit-application/src/main/java/org/dcsa/jit/Application.java

```diff
@@ -4,9 +4,11 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @ComponentScan("org.dcsa")
 @EntityScan("org.dcsa")
+@EnableJpaRepositories("org.dcsa")
```

Comment:
Might be good to move the scans to a configuration class, as we do in the other projects.